Neos.Flow/Classes/Command/DoctrineCommandController.php

```diff
@@ -495,8 +497,8 @@ public function migrationGenerateCommand(bool $diffAgainstCurrent = true, string
             $this->outputLine();
 
             if ($selectedPackage !== $choices[0]) {
-                /** @var Package $selectedPackage */
                 $selectedPackage = $packages[$selectedPackage];
+                /** @var Package $selectedPackage */
```

Comment:
I have no idea why this line was moved – it's not related and the previous version even makes more sense to me